src/main/kotlin/com/openlattice/linking/controllers/LinkingFeedbackController.kt

```diff
@@ -65,7 +67,7 @@ constructor(
                     "Cannot submit feedback for less than 2 entities or if no positively linking entity is provided")
         }
 
-        if(!Sets.haveEmptyIntersection(feedback.link, feedback.unlink)) {
+        if( Collections.disjoint( feedback.link, feedback.unlink )) {
             throw IllegalArgumentException("Cannot submit feedback with and entity being both linking and non-linking")
```

Comment:
this isn't your typo, but I think the exception below should be "an entity" instead of "and entity"